politeiawww/api/v1/api.md

```diff
@@ -1396,13 +1396,14 @@ Reply:
 
 Retrieve proposal and its details.
 
-**Routes:** `GET /v1/proposals/{token}`
+**Routes:** `GET /v1/proposals/{token}?version={proposalversion}`
```

Comment:
is it really the token being passed as a query param? if so I don't think it is encrypted, does not sound secure to me.

Reply:
This is the proposal censorship token. It has nothing to do with authentication tokens. Please, check the docs for further information.